src/filters/fs.rs

```diff
@@ -132,12 +132,14 @@ fn sanitize_path(base: impl AsRef<Path>, tail: &str) -> Result<PathBuf, Rejectio
     Ok(buf)
 }
 
-#[derive(Debug)]
-struct Conditionals {
-    if_modified_since: Option<IfModifiedSince>,
-    if_unmodified_since: Option<IfUnmodifiedSince>,
-    if_range: Option<IfRange>,
-    range: Option<Range>,
+/// Conditionals that define certain aspects of file serving. These are usually based on the headers of a request, though they can be manually constructed.
+#[allow(missing_docs)] // TODO add docs for these (consult @seanmonstar)
```

Comment:
Probably just makes sense to try and come up with a rough first draft of these comments, then people can actually review them.  Easier to give feedback, sometimes, than it is to create from scratch. :)

Reply:
I think from memory the reason I didn't do that is because I don't actually understand what these do yet. Any pointers?

Reply:
Well, these are the various HTTP headers that might come in on a request that could be pertinent to file serving.

So, for each of them, maybe a simple one-line explanation of what the header does and then a link to relevant docs, for example, the RFC that they were standardized in, or perhaps the Mozilla MDN docs for the given header.

Reply:
Thanks for the clarification, this should be fixed now!